Kickstarter-iOS/Views/Cells/BackerDashboardEmptyStateCell.swift

```diff
@@ -19,17 +26,51 @@ internal final class BackerDashboardEmptyStateCell: UITableViewCell, ValueCell {
         .~ UIImage(named: "icon--eye",
                    in: .framework,
                    compatibleWith: nil)
+      self.filledHeartIconImageView.isHidden = true
```

Comment:
Since we are using lenses for the other image view, can we change this line to:
`_ = self.filledHeartIconImageView
     |> UIImageView.lens.isHidden .~ true` ?